### PR TITLE
Confirm logged in as developer or die, nicer notices, nicer if's.

### PR DESCRIPTION
**Pull request by singularo**

### CODE REVIEW

File: dsh [hunk 1]
```diff
@@ -74,13 +74,13 @@ dsh_openshift_init() {
...
+    notice "Using OpenShift in Docker."
   fi
 
-  # Are we logged in?
-  if [ ! $(oc whoami) ]; then
-    # Prompt for login.
-    oc login
+  # Are we logged in as local developer, if not, bail!
+  if [[ $(oc whoami) != "developer" ]]; then
+    error "You are not logged in as developer, exiting for safety reasons."
```
Comment (pingers) [inline dsh]: I'd add "Please login to your local development OpenShift."

Comment (singularo) [reply dsh]: :+1: Done

File: dsh [hunk 2]
```diff
@@ -92,24 +92,26 @@ dsh_openshift_init() {
       oc annotate svc mysql-external "service.alpha.openshift.io/dependencies=[{\"name\": \"mariadb\", \"kind\": \"Service\"}]"
     fi
   else
-    echo "Not using Minishift, assuming mysql is available on port 3306."
```
Comment (pingers) [inline dsh]: Above this is a service check for mysql - could just `oc get svc mysql`?

Comment (singularo) [reply dsh]: Probably, don't have minishift to test, but.

**Comment by pingers** [comment]
Should probably fix the dsh_shell docker ps check... it changed upstream.
`if docker-compose -f ${DOCKER_COMPOSE_FILE} ps --filter "status=stopped" --services | grep web > /dev/null; then`

**Comment by pingers** [comment]
dsh status changed too...
https://github.com/universityofadelaide/shepherd-drupal-scaffold/blob/master/dsh#L83

**Comment by pingers** [comment]
:ship:

**Pull request merged**